internal/provider/resource_aci_endpoint_security_group_test.go

```diff
@@ -370,25 +367,29 @@ resource "aci_endpoint_security_group" "test" {
 `
 
 const testConfigFvESgAllDependencyWithFvAp = testConfigFvApMinDependencyWithFvTenant + `
+data "aci_system" "version" {
+  pod_id    = "1"
+  system_id = "1"
+}
+
 resource "aci_endpoint_security_group" "test" {
   parent_dn = aci_application_profile.test.id
   name = "test_name"
-  admin_state = "no"
   annotation = "annotation"
   description = "description_1"
   exception_tag = "exception_tag_1"
   intra_esg_isolation = "enforced"
   match_criteria = "All"
   name_alias = "name_alias_1"
   preferred_group_member = "exclude"
+  admin_state = provider::aci::compare_versions(data.aci_system.version.version,">=","5.2(1g)") ? "no" : null
 }
 `
 
 const testConfigFvESgResetDependencyWithFvAp = testConfigFvApMinDependencyWithFvTenant + `
 resource "aci_endpoint_security_group" "test" {
   parent_dn = aci_application_profile.test.id
   name = "test_name"
-  admin_state = "no"
```

Comment:
Why are we removing these?

Reply:
Introduced version_mismatch in default which takes care of this.